src/styles/button.st.css

```diff
@@ -0,0 +1,131 @@
+/**
+ * = BUTTON - button.st.css
+ *
+ * You should have enough sizes and hooks
+ * but you can always add your own custom classes for whatever you need or 
+ * wrap this button up inside other buttons pre configured for your usecases.
+ * 
+ * ========================================================================= */
+
+
+/* = Default project vars. */
+:import {
+  -st-from: "@actionishope/shelley/styles/default/project.st.css";
+  -st-named:
+    --spacing-unit;
+}
+
+/* = Shelley project scope and vars. */
+:import {
+  -st-from: "./project.st.css";
+  -st-default: Shelley;
+  -st-named:
+    /* = CSS3 Vars. */
+    --color-ui,
+    --color-ui-inverse,
+    --color-ui-raw,
+    --color-ui-raw-inverse,
+    --color-accent-1,
+    --color-accent-2,
+    --color-accent-3,
+    --color-accent-4;
+}
+
+/* = Import Button CSS API */
+:import {
+  -st-from: "@actionishope/shelley/components/Button/button.st.css";
+  -st-default: Button;
+  -st-named:
+    --bg-color,
+    --border-color,
+    --text-color;
+}
+
+/* = Import ButtonGroup CSS API */
+:import {
+  -st-from: "@actionishope/shelley/components/ButtonGroup/buttonGroup.st.css";
+  -st-default: ButtonGroup;
+  -st-named: --seperator-color;
+}
+
+/* = This is how we can scope the styles in Stylable. */
+@st-scope Shelley {
```

Comment:
I just called this project/theme Shelley but you could rename it to SweetPickings or create your own base to work with across projects.  :-)